.github/workflows/integration-tests.yaml

```diff
@@ -33,44 +33,82 @@ jobs:
         with:
           node-version: 18
 
+      - name: Install daemon process manager
+        run: npm install pm2 -g
+        working-directory: ./
+
+      - name: Install dependencies
+        run: yarn install
+        working-directory: ./
+
+      - name: Build packages
+        run: yarn build:packages
+        working-directory: ./
+
       - name: Start example application
-        run: |
-          # Install daemon process manager
-          npm install pm2 -g
-          yarn install
-          yarn build:packages
-          pm2 start "yarn start:example" --name web --wait-ready --listen-timeout 10000 # wait for 10 seconds
+        run: pm2 start "yarn start:example" --name web --wait-ready --listen-timeout 10000 # wait for 10 seconds
         working-directory: ./
 
-      - name: Start DMSS
-        id: dmss
+      - name: Pull docker images
+        run: docker-compose pull
+        working-directory: example/
+
+      - name: Capture output DMSS version
         run: |
-          docker-compose pull
           docker-compose run --rm dmss cat src/version.txt > dmss_version.txt
           dmss_version=$(tail -n 1 dmss_version.txt)
+          echo "dmss_version=$dmss_version" >> "$GITHUB_OUTPUT"
```

Comment:
The step is missing id, so the variable won't be accessible. Same for the job step below. So L27 and L28 will return empty values

Reply:
Good catch. Resolved